### PR TITLE
Uninstall Oracle JDK by default

### PR DESCRIPTION
**Pull request by MaxVigne**

### CODE REVIEW

File: tasks/setup-Debian.yml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 ---
+
+- name: Remove OracleJDK
+  apt:
+    name: "{{ item }}"
+    state: absent
+  with_items: "{{ oracle_jdk_packages }}"
```
Comment (pilou-) [inline tasks/setup-Debian.yml]: `name` parameter of `apt` package handles a list, there is no need to use a loop here.

File: tasks/setup-Debian.yml [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+  when: oracle_jdk_packages is defined and not keep_oracle_jdk
```
Comment (pilou-) [inline tasks/setup-Debian.yml]: `keep_oracle_jdk|bool` should be used

Comment (MaxVigne) [reply tasks/setup-Debian.yml]: OK i'm fixing it :+1:

File: README.md [hunk 3]
```diff
@@ -30,6 +30,13 @@ CA certificates can be added to the java keystore with the following variables:
       password: changeit
 ```
 
+```yaml
+    keep_oracle_jdk: false
+```
+
+This role uninstall OracleJDK by default. You can change the
```
Comment (pilou-) [inline README.md:37]: Should not `Debian based distributions only` added ?

Comment (MaxVigne) [reply README.md:37]: I agree :)

**Review by pilou- — APPROVED** [review]
I understand why these tasks are added in this role, using a common tasks `install Java` file in our common repo is an alternative too:
1. it would not increase delta with the original project
2. these tasks are Debian related only but original project isn't

**Pull request merged**